### PR TITLE
doc/rados: edit ceph-conf.rst (2 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit doc/rados/configuration/ceph-conf.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/configuration/ceph-conf.rst [hunk 1]
```diff
@@ -94,27 +94,26 @@ entirely.
 Skipping monitor config
 -----------------------
 
-Pass the option ``--no-mon-config`` to any process to skip the step that
-retrieves configuration information from the cluster monitors. This is useful
-in cases where configuration is managed entirely via configuration files, or
-when the monitor cluster is down and some maintenance activity needs to be
-done.
-
+The option ``--no-mon-config`` can be passed to any process in order to skip
```
Comment (anthonyeleven) [inline doc/rados/configuration/ceph-conf.rst:97]: `process` seems awkward here.  Maybe `command or daemon`?

File: doc/rados/configuration/ceph-conf.rst [hunk 2]
```diff
@@ -94,27 +94,26 @@ entirely.
...
+the step that retrieves configuration information from the cluster monitors.
```
Comment (anthonyeleven) [inline doc/rados/configuration/ceph-conf.rst:98]: possessive I think

File: doc/rados/configuration/ceph-conf.rst [hunk 3]
```diff
@@ -94,27 +94,26 @@ entirely.
...
+Skipping that retrieval step can be useful in cases where configuration is
```
Comment (anthonyeleven) [inline doc/rados/configuration/ceph-conf.rst:99]: that/this

File: doc/rados/configuration/ceph-conf.rst [hunk 4]
```diff
@@ -157,41 +156,41 @@ These sections include:
...
-section, the last value wins.
-
-Note that values from the local configuration file always take
-precedence over values from the monitor configuration database,
-regardless of which section they appear in.
+section, the last value specified takes precedence.
 
+Note that values from the local configuration file always take precedence over
+values from the monitor configuration database, regardless of which section
+they appear in.
```
Comment (anthonyeleven) [inline doc/rados/configuration/ceph-conf.rst:182]: the section in which

File: doc/rados/configuration/ceph-conf.rst [hunk 5]
```diff
@@ -157,41 +156,41 @@ These sections include:
...
 Metavariables
 =============
 
-Metavariables simplify Ceph Storage Cluster configuration
-dramatically. When a metavariable is set in a configuration value,
-Ceph expands the metavariable into a concrete value at the time the
-configuration value is used. Ceph metavariables are similar to variable expansion in the Bash shell.
+Metavariables dramatically simplify Ceph storage cluster configuration. When a
+metavariable is set in a configuration value, Ceph expands the metavariable
+into a concrete value at the time the configuration value is used. In this way,
```
Comment (anthonyeleven) [inline doc/rados/configuration/ceph-conf.rst:191]: s/into a concrete value//

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/configuration/ceph-conf.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/configuration/ceph-conf.rst:97]: Accepted.

File: doc/rados/configuration/ceph-conf.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/configuration/ceph-conf.rst:98]: s/cluster/cluster's/
Accepted.

File: doc/rados/configuration/ceph-conf.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/configuration/ceph-conf.rst:99]: I wondered about this as I was looking at it. Thanks for the benefit of your fresh eyes.

Accepted.

File: doc/rados/configuration/ceph-conf.rst (hunk 4 above)
Comment (zdover23) [reply doc/rados/configuration/ceph-conf.rst:182]: Well, okay.

File: doc/rados/configuration/ceph-conf.rst (hunk 5 above)
Comment (zdover23) [reply doc/rados/configuration/ceph-conf.rst:191]: Good one.

Accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/52470 - Reef backport
https://github.com/ceph/ceph/pull/52471 - Quincy backport

**Pull request merged**